### PR TITLE
Add timeouts to the pushover, hipchat, slack handlers

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by SpacePossum** [comment]
I'm not sure about the release/review cycle here, but if any of the maintainers got some pointers I'm happy to hear about it :D

**Comment by keradus** [comment]
ping @symfony/monologbundle 
who can we ask for help/directions to move this PR forward?
thanks !

**Comment by keradus** [comment]
e

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ MonologBundle
 =============
 
 The `MonologBundle` provides integration of the [Monolog](https://github.com/Seldaek/monolog)
-library into the Symfony2 framework.
+library into the Symfony2/3/4 framework.
```
Comment (fabpot) [inline README.md]: What about using just Symfony (that's what we've been doing elsewhere)? `into the Symfony framework`

Comment (SpacePossum) [reply README.md]: that makes a lot of sense :+1: updated

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request merged**